scrapers/mediafusion.py

```
@@ -0,0 +1,219 @@
+                    filename=parsed_data["file_name"],
+                    file_index=stream_data.get("fileIdx"),
+                    languages=parsed_data["languages"],
+                    resolution=parsed_data["metadata"].get("resolution"),
+                    codec=parsed_data["metadata"].get("codec"),
+                    quality=parsed_data["metadata"].get("quality"),
+                    audio=parsed_data["metadata"].get("audio"),
+                    source=source,
+                    catalog=["mediafusion_streams"],
+                    seeders=parsed_data["seeders"],
+                    announce_list=[
+                        tracker.lstrip("tracker:")
```

Ruff

Using `.strip()` with multi-character strings is misleading

(B005)